reference/resources/bgpconfig.md

```diff
@@ -46,7 +46,7 @@ spec:
 | name     | Unique name to describe this resource instance. Required. | Alphanumeric string with optional `.`, `_`, or `-`. | string |
 
 - The resource with the name `default` has a specific meaning - this contains the BGP global default configuration.
-- The resources with the name `node.<nodename>` contain the node-specific overrides, and will be applied to the node `<nodename>`. When deleting a node the BGPConfiguration resource associated with the node will also be deleted.
+- The resources with the name `node.<nodename>` contain the node-specific overrides, and will be applied to the node `<nodename>`. When deleting a node the BGPConfiguration resource associated with the node will also be deleted. Only Prefix Advertisements, Listen Port, Log Severity and Extensions can be overridden this way.
```

Comment:
Sorry, my bad from how I wrote this earlier in Slack, but I think we should align these field names with exactly how they're shown and capitalized in the following table.

Reply:
Extensions - what are they?  I don't see them in the table?